test/integration/testStreamRows.js

```diff
@@ -4,6 +4,7 @@
 var assert = require('assert');
 var async = require('async');
 var testUtil = require('./testUtil');
+const util = require("../../lib/util");
```

Comment:
do you need changes in this file?

Reply:
not now, new test are in separated file

Reply:
so let's revert all changes from this file